Runtime/StringEvents.cs

```diff
@@ -11,6 +11,8 @@ public class Cardholder
 {
     // cardholder's full name e.g John Doe. 
     public string name;
+
+    public Cardholder(string name) {this.name = name;}
```

Comment:
Other constructors in this file don't use single-line shorthand
```suggestion
    public Cardholder(string name)
    {
        this.name = name;
    }
```